### DISCUSSION TITLE
Durée d'activité de la session figée

### DISCUSSION THREAD

**Discussion by JeremieDENAEGEL** [witsa/synapps#795] [Q&A]
Bonjour,

La valeur "expirationdelay" de la session synapps semble rester figée à 1800 même sans bouger la souris.
De ce fait, la session synapps reste ouverte au delà de la durée d'activitée définie pour l'utilisateur.

Je lis la valeur sur une scène ouverte par un acteur écran de ma scène principale, est-ce la le problème ?

**Comment by manufitoussiwit** [comment]
Salut, comme pour les synoptiques, la durée d'expiration est restaurée à chaque fois que le REDY est sollicité. Donc à chaque fois qu'on charge une donnée par exemple.
